web-module/src/main/webapp/templates/celContextMenu/cel_cm_gallery_view/gal_edit.vm

```diff
@@ -4,8 +4,12 @@
 #else
 #set($galleryDocRef = $doc.documentReference)
 #end
-#set($l = "$services.url.getURL($galleryDocRef, 'edit')")
-#set($l = "${l}?language=${language}")
-#set($cmenu_pageLink = "${l}")
+#set($xredirect = "$!doc.getURL()")
+#if("$!xredirect" == '')
+#set($xredirect = '/')
+#end
+#set($xredirect = $escapetool.url("$!xredirect"))
+#set($l = $services.url.getURL($galleryDocRef, 'edit', "language=${language}"))
+#set($cmenu_pageLink = "${l}&xredirect=$!xredirect")
 window.location.href=&quot;${cmenu_pageLink}&quot;##
 #end
```

Comment:
newline at EOF missing

Reply:
no newline at EOF may be added because the result is a JavaScript inline output and a newline breaks the JS inline syntax.

Reply:
ok, this commit has added a newline though:
https://github.com/celements/celements-photo/pull/39/commits/747bf7afccab82456a1a3a86a5812d2973a9c3b7